roles/yoda_rulesets/defaults/main.yml

```diff
@@ -205,6 +205,7 @@ irods_arb_min_gb_free: 0
 irods_arb_min_percent_free: 5
 
 irods_anonymous_account_permit_addresses: []
+irods_user_max_connections: 3                          # Limit the number of iRODS connections per user
```

Comment:
When uploading a file in one tab, trying to log in another container tab, and running the main playbook I ran against the max connection limit. Otherwise I didn't hit the limit. Maybe set to 4?